### PR TITLE
STIJ-408 Add Pagination Hover States

### PR DESCRIPTION
**Pull request by donkeysbe**
## Description
Adds hover state to the pagination molecule

## Motivation and Context
https://district09.atlassian.net/browse/STIJ-408

## How Has This Been Tested?
On local env : gulp build

## Screenshots (if appropriate):
https://district09.atlassian.net/browse/STIJ-408?focusedCommentId=264401

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -0,0 +1,1157 @@
+# Changelog
```
Comment (zero2one) [inline docs/changelog.md]: What is the difference between the root CHANGELOG.md and this file?

**Review by zero2one — CHANGES_REQUESTED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (donkeysbe) [reply docs/changelog.md]: The root CHANGELOG.md gets copied to the docs folder by the gulpscript. If this build/compile script is also running on deploy maybe this particular file should be in gitignore ? Not sure though. @jeroendegloire can you elaborate?

Comment (jeroendegloire) [reply docs/changelog.md]: Yes docs/changelog.md has to be in gitignore

Comment (zero2one) [reply docs/changelog.md]: The docs.changelog.md file needs to be deleted from this PR

**Review by zero2one — CHANGES_REQUESTED** [review]

**Review by zero2one — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this style guide are documented here.
 
+## [5.0.16]
```
Comment (jeroendegloire) [inline CHANGELOG.md]:
```suggestion
## Unreleased
```

**Review by jeroendegloire — APPROVED** [review]

**Pull request merged**